app/utils/lava_log_parser.py

```diff
@@ -72,11 +75,47 @@ def run(log, boot, txt, html):
     else:
         boot_result_html = "<span class=\"warn\">{}</span>".format(boot_result)
 
+    # panic, error and warn are considered deprecated, but still here 
+    # for ensure that everything will works well in any case.
     formats = {
+        "emerg": "<span class=\"info\">{}</span>\n",
+        "alert": "<span class=\"alert\">{}</span>\n",
+        "crit": "<span class=\"alert\">{}</span>\n",
+        "err": "<span class=\"err\">{}</span>\n",
         "warning": "<span class=\"warn\">{}</span>\n",
+        "notice": "<span class=\"info\">{}</span>\n",
+        "info": "<span class=\"info\">{}</span>\n",
+        "debug": "<span class=\"debug\">{}</span>\n",
+        "panic": "<span class=\"alert\">{}</span>\n",
         "error": "<span class=\"err\">{}</span>\n",
+        "warn": "<span class=\"warn\">{}</span>\n"
     }
-    numbers = {"warning": 0, "error": 0}
+
+    kernel_log_levels = {
+        "0": "<span class=\"info\">{}</span>\n",
```

Comment:
Level 0 is the highest kind of error:
```c
#define KERN_EMERG      KERN_SOH "0"    /* system is unusable */
```
So maybe the same colour as "alert", but not "info".